src/TRestRawSignal.cxx

```diff
@@ -168,6 +171,18 @@ Short_t TRestRawSignal::operator[](Int_t n) {
 ///
 Double_t TRestRawSignal::GetData(Int_t n) const { return (Double_t)fSignalData[n] - fBaseLine; }
 
+///////////////////////////////////////////////
+/// \brief It returns a vector of the data with the
+/// baseline substracted
+///
+std::vector<Float_t> TRestRawSignal::GetData() const {
+    std::vector<Float_t> signalData(GetNumberOfPoints());
+
+    for (int i = 0; i < GetNumberOfPoints(); i++) signalData[i] = GetData(i);
```

Comment:
```suggestion
    for (int i = 0; i < GetNumberOfPoints(); i++) {
        signalData[i] = GetData(i);
    }
```

I think we should try to always use braces to avoid potential errors in the future.